### PR TITLE
Fix compilation issue

### PR DESCRIPTION
**Pull request by overlordtm**

### CODE REVIEW

File: generics.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package generics
+
+// NewGeneric reminds you that Go does not support generics
+func NewGeneric() {
```
Comment (hajimehoshi) [inline generics.go:4]: New is better: https://blog.golang.org/package-names

**Comment by tucnak** [comment]
This is bullshit and not fun.

File: generics.go (hunk 1 above)
Comment (edwellbrook) [inline generics.go:4]: yup, should be `New`

**Comment by GeertJohan** [comment]
:+1:

File: generics.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package generics
```
Comment (dmitshur) [inline generics.go:1]: There should be a package comment here.

**Pull request closed without merging**